libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Actions/BaseSendTaskModuleContinueResponse.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using AdaptiveExpressions.Properties;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Actions
```

Comment:
>Microsoft.Bot.Builder.Dialogs.Adaptive.Actions [](start = 10, length = 46)

Should we add these actions under a "Teams" namespace? e.g.: Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Actions?

Reply:
I wasn't sure about this.   The package here: https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/ has been released for five months, with no indication it is a "preview' library, and it does not have .Teams on the end of the namespace.